submissions/kasionio/friends-app/index.js

```diff
@@ -0,0 +1,113 @@
+let users;
+
+const fetchUsersData = async (url, retries) => {
+	try {
+		let response = await fetch(url);
+		let responseJson = await response.json();
+		return responseJson.results;
+	} catch (err) {
+		if (retries > 0) throw err;
+		return await fetchUsersData(url, retries - 1);
+	}
+}
+
+const initApp = async () => {
+	const API_URL = 'https://randomuser.me/api/?results=10&inc=name,picture,gender,location,dob,cell,email&noinfo';
+	const NUMBER_OF_FETCH_RETRIES = 5;
+	users = await fetchUsersData(API_URL, NUMBER_OF_FETCH_RETRIES);
+	renderUserCards(users);
+	hideLoader();	
+}
+
+const cardsContainer = document.createDocumentFragment();
+const userCards = document.querySelector('#user-cards');
+
+const createUserCard = user => {
+	let userDiv = document.createElement('div');
+	userDiv.classList.add('user-card');
+	userDiv.innerHTML = `
+     <div class="name" id='fullName'>${user.name.title} <span id='firstName'>${user.name.first}</span> ${user.name.last}</div>
+     <img class="img" src=${user.picture.large} alt='avatar'>
+     <div class="gender">${user.gender}</div>
+     <div class="location">${user.location.country}</div>
+     <div class="age"><span id="age">${user.dob.age}</span> y.o.</div>
+     <a class='tel' href="tel:+${user.cell}">${user.cell.replace(/[^\d]/g, '')}</a>
+     <a class='email' href="mailto:${user.email}">Send email</a>
+ `;
+	cardsContainer.appendChild(userDiv);
```

Comment:
Well... function called `create`, not `append`.

So you should return `div` with a user and then append them all in `renderUserCards` at one step